server.go

```diff
@@ -8,20 +8,26 @@ import (
 	"net/http"
 	"bytes"
 	"strings"
-
-	"github.com/clarifai/clarifai-go"
 	"github.com/gin-gonic/gin"
+	"os"
+	"net/url"
 )
 
 const (
 	clarifaiApi = "https://api.clarifai.com/v2/models/aaa03c23b3724a16a56b629203edc62c/outputs"
+	clarifyAuth = "https://api.clarifai.com/v1/token"
 )
 
 type Server struct {
 	app    *gin.Engine
-	client *clarifai.Client
 }
 
+type AccessToken struct {
+	Token string `json:"access_token"`
+	ExpiresIn int `json:"expires_in"`
```

Comment:
As I mentioned above, it is not used yet, but it will be, so I just decided to get all the information we need for future logic, later on, we should only request the token after expires_in time passes